runner/ansible/roles/1.4.1.monitor_interval/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '1.4.1.monitor_interval'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    if cibadmin -Q --xpath "//primitive[@type='fence_azure_arm']/@type" > /dev/null 2>&1; then
+        cibadmin -Q --xpath "//primitive[@type='fence_azure_arm']/operations/op [@name='monitor']" | grep -oP 'interval="\K[^"]+' || echo ""
+    else
+        # Azure Fence Agent is not configured. We echo expected values just to fulfill the check
+        echo 3600
```

Comment:
This `3600` should we `expected[test_id]` I guess

Reply:
As per our talk. I will cllaps all 1,4.1 tests into a single test with pass/fail return.